modules/userId/eids.js

```diff
@@ -220,3 +220,23 @@ export function createEidsArray(bidRequestUserId) {
   }
   return eids;
 }
+
+/**
+ * @param {SubmoduleContainer[]} submodules
+ */
+export function buildEidPermissions(submodules) {
+  let eidPermissions = [];
+  submodules.filter(i => utils.isPlainObject(i.idObj) && Object.keys(i.idObj).length)
+    .forEach(i => {
+      Object.keys(i.idObj).forEach(key => {
+        if (utils.deepAccess(USER_IDS_CONFIG, key + '.source')) {
+          let eidPermission = { source: USER_IDS_CONFIG[key].source };
+          if (utils.deepAccess(i, 'config.bidders') && Array.isArray(i.config.bidders)) {
+            eidPermission.bidders = i.config.bidders;
+          }
+          eidPermissions.push(eidPermission);
```

Comment:
I don't think we should be adding anything to the eidPermissions array unless there are actually permissions to be added. i.e. if there's no `bidders` array specified, why do we need an entry at all?

Reply:
Previously entry contained bidders: ['*'] if every bidder allowed, but then decided to omit bidders in this case.
Anyway, I will omit this entry entirely